jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/impl/MessageDescImplTest.java

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Shouldn't this test class be located under a package `org.jbpm.kie.services.impl.model` just to follow same package structure for the class to test?